css/pages/_journal.scss

```diff
@@ -0,0 +1,59 @@
+.journals {
+	margin-top:30px;
+}
+.tab-nav {
+	width:15%;
+	float:left;
+	margin-top:40px;
+}
+.tab {
+	padding:10px;
+	cursor:pointer;
+	font-weight:bold;
+}
+.tab:hover {
+	background-color:$primary-red;
+	color:#FFF;
+}
+.tab-content {
+	width:85%;
+	float:right;
+}
+.tab-title {
+	color:$primary-grey;
+	text-transform:uppercase;
+	text-align:center;
+	font-size:1.1em;
+	letter-spacing:.1em;
+}
+.cover-img {
+	width:30%;
+	float:left;
+	padding:20px;
+}
+.author {
```

Comment:
This is a pretty generic class name, either make it more specific or nest it inside a parent container so it's scoped to the page